docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,544 @@
+    - `General.BootstrapMethod` - Set this value to `none`. Because the system channel is no longer required, the `orderer.yaml` file on each orderer needs to be configured with `BootstrapMethod: none` which means that no genesis block is required or used to start up the ordering service.
+    - `Admin.ListenAddress` - The orderer admin server address (host and port) that can be used by the `osnadmin` command to configure channels on the ordering service. This value should be a unique `host:port` combination to avoid conflicts.
+    - `Admin.TLS.Enabled:` - This value must be set to `true`.
+    - `Admin.TLS.PrivateKey:` - The path to and file name of the orderer private key issued by the TLS CA.
+    - `Admin.TLS.Certificate:` - The path to and file name of the orderer signed certificate issued by the TLS CA.
+    - `Admin.TLS.RootCAs:`  - The path to and file name of the TLS CA Root cert `tls-ca-cert.pem`.
+    - `Admin.TLS.ClientAuthRequired:` This value must be set to `true`. Note that while Mutual TLS is required for all operations on the orderer Admin endpoint, the entire network is not required to use Mutual TLS.
+    - `Admin.TLS.ClientRootCAs:` - The path to and file name of the TLS CA Root cert `tls-ca-cert.pem`.
+    - `ChannelParticipation:Enabled` - Set this value to `true` to enable this feature on the orderer.
+- Restart each ordering node.
+- Put the [system channel into maintenance mode](../kafka_raft_migration.html#entry-to-maintenance-mode) using the same process for the Kafka to Raft migration. This action stops new channel creation transactions from coming in.
+- Remove the system channel from the set of orderers, one by one. Care must be taken to ensure that the remaining orderers can still function and reach consensus. This operation can be staggered, such that depending on the fault tolerance setup of the respective channels, no channel down time is experienced. If an application channel can tolerate one server offline, you should still be able to submit transactions to the channel, via the other orderers that are not undergoing the removal at that time.  Use the `osnadmin channel list` command to view the channels that this orderer is a member of:
+    ```
+    export TLS_CA_ROOT_CERT=../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/cacerts/tls-ca-cert.pem
+    export OSN_TLS_SIGN_CERT=../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/signcerts/cert.pem
+    export OSN_TLS_PRIVATE_KEY=../config/organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls/keystore/key.pem
+
+    osnadmin channel list -o [ORDERER_ADMIN_LISTENADDRESS] --ca-file $TLS_CA_ROOT_CERT --client-cert $OSN_TLS_SIGN_CERT --client-key $OSN_TLS_PRIVATE_KEY
+    ```
```

Comment:
I am confused by the naming of these variables. 
This `--ca-file $TLS_CA_ROOT_CERT` is the **server's** ROOT CA, correct? 
However, these two are the certificate and private key of the **admin client**, right?
`--client-cert $OSN_TLS_SIGN_CERT --client-key $OSN_TLS_PRIVATE_KEY`

So I think the naming should be revised - 
TLS_CA_ROOT_CERT -> OSN_TLS_CA_ROOT_CERT
OSN_TLS_SIGN_CERT -> ADMIN_TLS_SIGN_CERT
OSN_TLS_PRIVATE_KEY -> ADMIN_TLS_PRIVATE_KEY 

In addition, the folder structure above may be revised to clearly reflect the division between "server" crypto material and "client" crypto material. 

Is the tutorial using shared material for simplicity?

Note that the same client can be used to command all three OSNs!